.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.9.3
```

Comment:
Not sure how this happened, but dev wasn't initially synced with master before this release.